pkgs/tools/security/tessen/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, stdenvNoCC
+, fetchFromSourcehut
+, pass
+, bemenu
+, passBackend ? pass
+, picker ? bemenu
+, wtype
+, wl-clipboard
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "tessen";
+  version = "unstable-2022-08-04";
+
+  src = fetchFromSourcehut {
+    owner = "~ayushnix";
+    repo  = pname;
+    rev = "8758a09345f6eef24764de4a0efad737f12562c8";
+    sha256  = "sha256-U6obXpYzIprOJ+b3QiE+eDOq1s0DYiwM55qTga9/8TE=";
+  };
+
+  propagatedUserEnvPkgs = [ passBackend picker wtype wl-clipboard ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D tessen $out/bin/tessen
+    install -Dm644 man/* -t $out/share/man/tessen
+    install -Dm644 config $out/etc/xdg/tessen/config
```

Thats a really unusual directory. How is that to be used?

---

The `config` file is supposed to be  `~/.config/tessen/config`, which the closest thing I've gotten to is `/etc/xdg/`

---

```suggestion
    install -Dm644 config $out/share/tessen/config
```
I would recommend something like